Longest_Increasing_Subsequence/LIS.kt

```diff
@@ -0,0 +1,51 @@
+import java.util.*
```

Comment:
File name should be `Longest_Increasing_Subsequence.kt`

Reply:
I do so because All other files have the same name LIS. If you still want I can change.

Reply:
Done